iOS/Shimmer/source/Shimmer/ApiDefinition.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using CoreAnimation;
+using Foundation;
+using ObjCRuntime;
+using UIKit;
+
+namespace Shimmer
+{
+    // @protocol FBShimmering <NSObject>
+    [Protocol, Model]
+    [BaseType(typeof(NSObject), Name = "FBShimmering")]
+    interface Shimmering
+    {
+        // @required @property (getter = isShimmering, assign, readwrite, nonatomic) BOOL shimmering;
+        [Abstract]
+        [Export("shimmering")]
+        bool IsShimmering { [Bind("isShimmering")] get; set; }
+
+        // @required @property (assign, readwrite, nonatomic) CFTimeInterval shimmeringPauseDuration;
+        [Abstract]
+        [Export("shimmeringPauseDuration")]
+        double ShimmeringPauseDuration { get; set; }
+
+        // @required @property (assign, readwrite, nonatomic) CGFloat shimmeringAnimationOpacity;
+        [Abstract]
+        [Export("shimmeringAnimationOpacity")]
+        nfloat ShimmeringAnimationOpacity { get; set; }
+
+        // @required @property (assign, readwrite, nonatomic) CGFloat shimmeringOpacity;
+        [Abstract]
+        [Export("shimmeringOpacity")]
+        nfloat ShimmeringOpacity { get; set; }
+
+        // @required @property (assign, readwrite, nonatomic) CGFloat shimmeringSpeed;
+        [Abstract]
+        [Export("shimmeringSpeed")]
+        nfloat ShimmeringSpeed { get; set; }
+
+        // @required @property (getter = shimmeringHighlightLength, assign, readwrite, nonatomic, setter = setShimmeringHighlightLength:) CGFloat shimmeringHighlightWidth;
+        [Abstract]
+        [Export("shimmeringHighlightWidth")]
+        nfloat ShimmeringHighlightWidth { [Bind("shimmeringHighlightLength")] get; [Bind("setShimmeringHighlightLength:")] set; }
```

Comment:
According to Obj-C headers, this property is now deprecated and replaced with `shimmeringHighlightLength` property.